caffe2/onnx/backend.cc

```diff
@@ -244,6 +256,9 @@ OnnxAttributes::OnnxAttrToCaffe2Arg(
     const auto &attr = rewritten_onnx_attrs_.count(kv.first)
                            ? rewritten_onnx_attrs_.at(kv.first)
                            : (*kv.second);
+    if (mapper(attr.name()).empty()) {
```

Comment:
this is not needed anymore

Reply:
For Upsample, we still need it: https://github.com/caffe2/caffe2/blob/master/caffe2/onnx/backend.cc#L319

If we just want to remove an attribute during onnx to caffe2 conversion, probably this handling is the easiest.

Reply:
alright add a TODO to create another explicit list of attributes that we want to remove, like `_blacklist_caffe2_args` in the frontend

Reply:
Sure, I will do that.